platform/core/src/services/ToolBarService/ToolBarService.js

```diff
@@ -27,6 +27,15 @@ export default class ToolBarService {
     return this.buttons;
   }
 
+  getActiveTools() {
+    return Object.keys(this.buttons).filter(key => {
+      const button = this.buttons[key];
+      if (button && button.props && button.props.isActive) {
+        return button;
+      }
+    });
```

Comment:
I think this is fine for now, but we definitely need to think on this post the 6th. We shouldn't be iterating over button props to determine which tools are active.

Reply:
CC: @JamesAPetts

Reply:
Yeah, i think the whole active tools logic should be updated in the near future when we move to a single state in toolbar manager and use the manager to push / update tools state instead